chapter-10-workload-identities/dbserver/README.md

```diff
@@ -63,9 +63,12 @@ GRANT SELECT, INSERT, UPDATE, DELETE ON products TO demoapi;
 
 ## X509 SVID Downloads
 
```

Comment:
The pattern of using both init and sidecar containers is mentioned in paragraph 3 of page 204 in the Production Kubernetes book, and adopting that simplifies the Postgres + SPIRE deployment.